gammapy/background/tests/test_energy_offset_array.py

```diff
@@ -0,0 +1,24 @@
+from ..energy_offset_array import EnergyOffsetArray
+from gammapy.data import DataStore
+from gammapy.data import EventList
+from ...datasets import gammapy_extra
+from ...utils.testing import requires_dependency, requires_data
+import numpy as np
```

Comment:
Put the numpy import further up.
We try to consistently use this order of imports:
* Python standard library
* External Python packages (like Numpy in this case)
* Gammapy relative imports